client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -27,6 +29,28 @@ interface Props {
 
 export default function Login({ handleSubmit }: Props): JSX.Element {
   const classes = useStyles();
+  const { updateLoginContext } = useAuth();
+
+  const hardCodedData = {
```

Comment:
In the future, when we have mock objects please consider giving descriptive variable names